src/main/menu.js

```diff
@@ -13,6 +13,9 @@ import { checkForUpdates } from './lib/updater';
 import { getMainWindow } from './utils/window';
 import i18n, { languages } from '../shared/i18n';
 import pkg from '../../package.json';
+import electronContextMenu from 'electron-context-menu';
+
+electronContextMenu();
```

Comment:
This is something I'm not sure if you will agree with. But I found for myself having `Inspect` option in development as essential when tweaking the UI. Using [this module](https://github.com/sindresorhus/electron-context-menu) was the easiest way I now of getting that menu there.

It adds also `Cut`, `Copy` and `Paste` for text, whenever it would suit and there isn't a Buttercup menu overriding it. I totally understand if you don't want to have these, please tell me what you think.

Reply:
I'll let @sallar check this one out. The Cut/Copy/Paste sounds good to me..

Reply:
@hugo-cardenas @perry-mitchell I like it too. If it doesn't interfere with other menus that we have

Reply:
It does not interfere, actually it is there unless your own menus overwrite it. This is due that I just added it as a base here, and whenever you would have a menu it just replaces it. If you are ok with this, good, it was just the least effort alternative.

If you would even like to merge it with your own menu, I could also do that, probably.